tests/test_per.py

```diff
@@ -91,6 +91,11 @@ def test_integer(self):
             "    b INTEGER (-10000..704000000000000001), "
             "    c BOOLEAN "
             "} "
+            "T ::= SEQUENCE { "
+            "  a BOOLEAN, "
+            "  ..., "
+            "  b NULL "
+            "} "
```

Comment:
The `T` sequence should be moved to another test as it does not test integers. Move it to the sequence test.

Reply:
Oh, you're right, I overlooked that. Is changed now.